plugins/ltac/rewrite.mli

```diff
@@ -62,7 +62,7 @@ type rewrite_result =
 
 type strategy
 
-val strategy_of_ast : (glob_constr_and_expr, raw_red_expr) strategy_ast -> strategy
+val strategy_of_ast : interp_sign -> (glob_constr_and_expr, glob_red_expr) strategy_ast -> strategy
```

Comment:
Can you confirm that we can backport this PR despite this API change?

Reply:
The probably that this function is used and produced an incompatibility is weak, but (unless there is a strong demand for having it in 8.12) it can also go in 8.13.

Reply:
@ejgallego Any opinion?

Reply:
Minor very API breakage vs bug fix ; I dunno.

The API breakage is not really important and I agree that it's unlikely it has impact on the wild; so I'd go with the opinion of the maintainers on this one [IMO they can assess if the bug is important enough as to place it in 8.12]

Reply:
Not a critical bug, I'd favour API stability even if very unlikely to be used anywhere.

Reply:
OK, not backporting then. Thanks for the feedback all!